dll/directx/dsound_new/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+
+spec2def(dsound.dll dsound.spec ADD_IMPORTLIB)
+
+list(APPEND SOURCE
+    capture.c
+    capturebuffer.c
+    classfactory.c
+    devicelist.c
+    directsound.c
+    dsound.c
+    enum.c
+    misc.c
+    notify.c
+    primary.c
+    property.c
+    regsvr.c
+    secondary.c
+    stubs.c
+    precomp.h
+    resource.h)
+
+add_library(dsound MODULE
+    ${SOURCE}
+    version.rc)
+
+set_module_type(dsound win32dll)
+target_link_libraries(dsound dxguid uuid)
+add_importlibs(dsound winmm advapi32 msvcrt kernel32 ntdll)
```

Comment:
add ``ole32`` here, I suppose

Reply:
Indeed! After adding `ole32` it does no longer spam about `CoTaskMemFree@4`! By the same principle I added other needed importlibs into `add_importlibs()` section: `ksuser`, `setupapi` and `user32`. And now compilation succeeds! Thanks for spotting! :smile: